<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>prismabaserect</class>
 <widget class="QDialog" name="prismabaserect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Prisma Base Rectangular</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(239, 41, 41);</string>
  </property>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>113</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Dibujar-Borrar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Reflexion Vertical</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="boxXinicio">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>280</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 238, 236);</string>
   </property>
   <property name="placeholderText">
    <string>Horizontal</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>100</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Rotar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>340</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Trasladar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Reflexion Horizontal</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="boxGrados">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>50</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 238, 236);</string>
   </property>
   <property name="placeholderText">
    <string>Grados</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="boxYinicio">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>230</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 238, 236);</string>
   </property>
   <property name="placeholderText">
    <string>Vertical</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
